src/style.css

```diff
@@ -1,3 +1,64 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+
+  --blue: #00acdc;
```

Comment:
Variable name should describe what it does, for example if the active color changes to `red` you have to change it in all places